src/components/search-form/search-form.js

```diff
@@ -339,10 +440,11 @@ class SearchForm extends Component {
           {Filters && (
             <div role="tablist">
               {this.renderTagFilter()}
+              {searchType === searchTypes.PACKAGES && this.renderAccessTypesFilter()}
               <Filters
                 activeFilters={combinedFilters}
                 onUpdate={this.handleUpdateFilter}
-                disabled={searchByTagsEnabled}
+                disabled={searchByTagsEnabled || searchByAccessTypesEnabled}
```

Comment:
Cover `searchByTagsEnabled || searchByAccessTypesEnabled` in some variable cause you use it in this file three times